docs/source/modin/supported/series_supported.rst

```diff
@@ -19,7 +19,7 @@ Attributes
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``array``                   | N                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``at``                      | N                               |                                                    |
+| ``at``                      | P                               | ``N`` for set with MultiIndex                      |
```

Comment:
You missed changing the `N` to `Y` for `Series.iat`

Reply:
Done.